midenc-session/src/libs.rs

```diff
@@ -23,8 +22,6 @@ use crate::{
 
 pub static STDLIB: LazyLock<Arc<CompiledLibrary>> =
```

Comment:
We should probably ditch `STDLIB` in favor of `StdLibrary::default().as_ref().clone()` as well for consistency.